main.js

```diff
@@ -33,11 +33,10 @@ let mBusDevices = {};
 let deviceCommunicationInProgress = false;
 let mbusMaster;
 
-let connected = null;
+let connected = false;
```

Comment:
Formally you would need to read "connection" at beginning to know what was the value. There may be "crash" cases where the state value was left as "true" or such and then at start you would not wrte the correct value. Whats that problematic with the debug message? In worst case remove the logging :-) Other question is why this happens at all because the value is checked before and once it was set to false it shpuld do nothing ... strange ...

Reply:
This is basically completely irrelevant. Since main calls setConnected(false) this results in a debug message as "connected" is initialized with null/undefined (because false != null). As I based the wireless mbus adapter partially on your code, I was irritated why there was the first debug message... but as I said this is irrelavant.

Reply:
understood, as described it is set to make sure value is correct (for the case it was left over with "true" and e.g. now there is an connection error ... else this would show incorrectly